x-pack/plugins/security_solution/scripts/run_cypress/parallel_serverless.ts

```diff
@@ -464,6 +464,10 @@ ${JSON.stringify(cypressConfigFile, null, 2)}
                 if (cloudHandler instanceof CloudHandler) {
```

Comment:
nit: I'd move a cleanup task to a function for better readability and modularability.

Reply:
I agree, however I am still learning this code structure, and the next task I am working on is generalizing functionality in the parallel and parallel_serverless scripts out into a common file.  I think that PR would be a better place for restructuring something like this call.